### PR TITLE
Faster liveview trimming

### LINKED ISSUE #68: Fix liveview performance
Unfortunately, the changes in https://github.com/jsdelivr/globalping-cli/pull/67 made the CLI extremely slow. 

Before:
```
$ time go run main.go http https://www.jsdelivr.com --from Prague --full
real    0m1.329s
user    0m0.372s
sys     0m0.118s
```

After:
```
$ time go run main.go http https://www.jsdelivr.com --from Prague --full
real    0m5.059s
user    0m4.163s
sys     0m0.174s
```

The difference is clearly noticeable and reproducible when switching between `ac64ed505f00f9dd374b6b234ea3bedd8463452d` and `82a515482980685d8089bc75eb58b37893b720a7` revisions. From the diff, this line seems most suspicious: https://github.com/jsdelivr/globalping-cli/pull/67/files#diff-a11c808cf8905bf8e77f93433ad981e30426c8716b5f8854b1e2f6066adfab7fR47

### PR DESCRIPTION
**Pull request by didil**
Fixes #68 
Use single call to runewidth.StringWidth to optimize performance

### CODE REVIEW

**Comment by jimaek** [comment]
Can you please run the same test as Martin to see the perf difference?

**Comment by didil** [comment]
Before the PR I get:
```
real    0m4.698s
user    0m3.435s
sys     0m0.239s
```
With this PR I get:
```
real    0m1.567s
user    0m0.330s
sys     0m0.233s
```

File: view/view_test.go [hunk 1]
```diff
@@ -369,7 +369,7 @@ LOREM IPSUM LOREM IPSUM LOREM IPSUM`
 
 	res := trimOutput(output, 84, 7)
 
-	expectedRes := `LOREM  IPSUM ♥ LOREM IPSUM LOREM IPSUM LOREM IPSUM LOREM IPSUM LOREM IPSUM LOREM
+	expectedRes := `LOREM  IPSUM ♥ LOREM IPSUM LOREM IPSUM LOREM IPSUM LOREM IPSUM LOREM IPSUM LOREM I
```
Comment (MartinKolarik) [inline view/view_test.go:372]: I'm guessing this change is because it won't handle some emoji-related edge cases with 100% precision now?

**Comment by MartinKolarik** [comment]
Performance seems to be back to normal 👍

**Review by MartinKolarik — APPROVED** [review]

**Comment by didil** [comment]
found a bug (crashes on chinese language website https://www.zhihu.com/signin due to the approximation). working on fixing it

**Comment by didil** [comment]
- fixed edge case with full line of chinese characters
- increased error margin from 2 to 4 spaces for width and length

**Pull request merged**